packages/block-editor/src/utils/dom.js

```diff
@@ -8,12 +8,12 @@
  *
  * @return {Element} Block DOM node.
  */
-export function getBlockDOMNode( clientId, scope = document ) {
```

Comment:
The JSDoc should have been updated as well. There's currently a lint warning:

```
[0] /Users/andrew/Documents/Code/gutenberg/packages/block-editor/src/utils/dom.js
[0]   7:0  warning  @param "scope" does not match an existing function parameter  jsdoc/check-param-names
```

Reply:
How do you need the lint warning? Why are we not throwing an error for incomplete docs? Shall I create a PR to rectify this?

Reply:
> Why are we not throwing an error for incomplete docs?

We used to. Part of the transition to `eslint-plugin-jsdoc` downgraded them to warnings (because this is how they're defined in the recommended set).

See: #18458

We have some other warnings now as well, some caused by TypeScript syntaxes considered as invalid (see https://github.com/WordPress/gutenberg/issues/18458#issuecomment-576749575).